Source_codes/SPO2/arduino-pulsoxy/UART.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * UART.c
+ *
+ * Created: 14.03.2018 12:21:11
+ *  Author: Moritz Klimt
+ */ 
+
+#ifndef F_CPU
+#define F_CPU 16000000UL
+#endif
+
+#include <avr/io.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+#include <avr/interrupt.h>
+#include <util/delay.h>
+
+void init_UART_115200_8N1()
+{
+	UCSR0A |= (1 << U2X0);							// Set the "Double Transmission Speed" bit in order to use a more precise timing mode for 115200 Baud
+	UCSR0B |= ((1 << RXEN0) | (1 << TXEN0));		// Enable receive and transmit mode which again configure the respective pins accordingly
+	UCSR0C |= ((1 << UCSZ01) | (1 << UCSZ00));		// Set 8-bit data mode
+	UBRR0L = 16;									// Set BAUD rate to 115200;
```

Comment:
where does this new value come from? prior value was 206, change is undocumented

Reply:
Good catch! Must have been a dirty working directory. This change was not intentional. The files here should match the ones from the original project. The original value is `207`.